### ISSUE TITLE
[Tabs] Make disabled tabs focusable

### ISSUE THREAD

**Issue by a11ymiko** [sl-design-system/components#3623]
### Provide a general summary of the issue here

Right now tabs use `disabled` attribute for disabling tab for keyboard and mouse. Options disabled with this method are not accessible via keyboard with standard navigation.  

It's better to use only `aria-disabled` and make disabled tabs accessible via keyboard. Disabled options still shouldn't be possible to be activated by mouse or keyboard but they will be focusable with standard keyboard navigation. Then user of assistive technologies could get information about every tab much easier. 

This approach is recommended by W3C: https://www.w3.org/WAI/ARIA/apg/practices/keyboard-interface/#kbd_disabled_controls

### 👤 Your name

Kacper

### 🧢 Your Product/Team

SLDS
